baselines/powerplatform.md

```diff
@@ -420,42 +410,25 @@ response to Section 3 of [Executive Order (EO) 14028, *Improving the
 Nation’s
 Cybersecurity*](https://www.federalregister.gov/documents/2021/05/17/2021-10460/improving-the-nations-cybersecurity):
 
-- Consumer Financial Protection Bureau (CFPB)
-
-- Department of the Interior (DOI)
-
-- National Aeronautics and Space Administration (NASA)
-
+- The MITRE Corporation
```

Comment:
Observation:  Teams has this {MITRE & Sandia} Aad does not.

Reply:
added new front-matter and back matter and criticality.